tests/integration/telemetry/tracing/tracing_test.go

```diff
@@ -71,7 +71,7 @@ func TestProxyTracing(t *testing.T) {
 					return errors.New("cannot find expected traces")
 				}
 				return nil
-			}, retry.Delay(3*time.Second), retry.Timeout(40*time.Second))
+			}, retry.Delay(3*time.Second), retry.Timeout(80*time.Second))
```

Comment:
question: what is the default time out for running these tests in CI?

Reply:
It is 30s by default: https://github.com/istio/istio/blob/d6d298327117385a51a51a3a4d65f96961006964/pkg/test/util/retry/retry.go#L26